test/manifest-v7.6.js

```diff
@@ -0,0 +1,205 @@
+          'type': 'geojson',
+          'url': '/files/rohan_v2.geo.json',
+          'legacy_default': false,
+        },
+        {
+          'type': 'topojson',
+          'url': '/files/rohan_v2.topo.json',
+          'legacy_default': true,
+          'meta': {
+            'feature_collection_path': 'regions',
+          },
+        },
+      ],
+      'fields': [
+        {
+          'type': 'id',
+          'id': 'wikidata',
+          'label': {
+            'de': 'Wikidata-Kennung',
+            'en': 'Wikidata identifier',
+            'zh': '维基数据标识符',
+          },
+        },
+        {
+          'type': 'property',
+          'id': 'label_en',
+          'label': {
+            'de': 'name (en)',
+            'en': 'name (en)',
+            'zh': '名称 (en)',
+          },
+        },
+      ],
+      'legacy_ids': [
+        'Rohan',
+        'Rohan Kingdoms',
+      ],
+      'layer_name': {
+        'en': 'Rohan Kingdoms',
+        'de': 'Rohan',
+        'zh': '洛汗',
+      },
+    }, {
+      'layer_id': 'shire',
+      'created_at': '1532-12-25T18:45:32.389979',
+      'attribution': [{
+        'label': {
+          'en': 'The Silmarillion',
+        },
+      }],
+      'formats': [
+        {
+          'type': 'geojson',
+          'url': `/files/shire_v2.geo.json`,
+          'legacy_default': true,
+        },
+        {
+          'type': 'topojson',
+          'url': '/files/shire_v2.topo.json',
+          'legacy_default': false,
+
+        },
+      ],
+      'fields': [
+        {
+          'type': 'id',
+          'id': 'wikidata',
+          'label': {
+            'de': 'Wikidata-Kennung',
+            'en': 'Wikidata identifier',
+            'zh': '维基数据标识符',
+          },
+        },
+        {
+          'type': 'property',
+          'id': 'label_en',
+          'label': {
+            'de': 'name (en)',
+            'en': 'name (en)',
+            'zh': '名称 (en)',
+          },
+        },
+        {
+          'type': 'property',
+          'id': 'label_ws',
+          'label': {
+            'de': 'name (ws)',
+            'en': 'name (ws)',
+            'zh': '名称 (ws)',
+          },
+        },
+      ],
+      'legacy_ids': [
+        'Shire',
+        'Shire regions',
+        'Shire Regions',
+      ],
+      'layer_name': {
+        'en': 'Shire regions',
+        'de': 'Auenland',
+        'zh': '夏爾',
+      },
+    },
+  ],
+};
+
+tap('>=7.6 tests', t => {
+  const catalogue = generateCatalogueManifest({
+    version: 'v7.6',
+    tileHostname: 'tiles.maps.elstc.co',
+    vectorHostname: 'vector.maps.elastic.co',
+  });
+  t.false(catalogue, '7.6 catalogue should not exist');
+
+  // It is not necessary to test different hostnames, since URLs in manifest are relative
+  const vector = generateVectorManifest(sources, {
+    version: 'v7.6',
+    hostname: 'vector.maps.elastic.co',
+    fieldInfo: fieldInfo,
+  });
+  t.deepEquals(vector, vectorExpected, 'v7.6 vector manifest');
+  t.end();
```

Comment:
Not sure if this is needed or helpful, but maybe we could check here not just `7.6` but future `v7` releases and that it fails for `v8` (so we can ensure we control that correctly later this year).

```js
tap('>=7.6 tests', t => {

  ['v7.6', 'v7.8', 'v7.8.1'].forEach(version => {
    const catalogue = generateCatalogueManifest({
      version: version,
      tileHostname: 'tiles.maps.elstc.co',
      vectorHostname: 'vector.maps.elastic.co',
    });
    t.false(catalogue, version + ' catalogue should not exist');

    // It is not necessary to test different hostnames, since URLs in manifest are relative
    const vector = generateVectorManifest(sources, {
      version: version,
      hostname: 'vector.maps.elastic.co',
      fieldInfo: fieldInfo,
    });
    t.deepEquals(vector, vectorExpected, version + ' vector manifest');
  });

  //v8 vector manifest should fail at this moment
  tap('Check that v8.0 fails', function (t) {
    t.throws(function () {
      generateVectorManifest(sources, {
        version: 'v8.0',
        hostname: 'vector.maps.elastic.co',
        fieldInfo: fieldInfo,
      });
    }, new Error('Unable to get a manifest for version 8.0.0'), 'throws assert error');
  });

  t.end();
});
```